test/parallel/test-process-env-allowed-flags-are-documented.js

```diff
@@ -46,7 +46,7 @@ const conditionalOpts = [
       return ['--openssl-config', '--tls-cipher-list', '--use-bundled-ca',
               '--use-openssl-ca' ].includes(opt);
     } },
-  { include: common.hasFipsCrypto,
+  { include: process.config.variables.openssl_is_fips,
```

Comment:
Can you add an explaining comment? I would also suggest updating the `common.hasFipsCrypto` entry in `test/common/README.md`.

Reply:
I've added a comment to this line now.  About adding a comment to common.hasFipsCrypto in the README, should that be something along the lines of what I added in f1e2dc6?